CMakeLists.txt

```diff
@@ -90,6 +90,9 @@ option (COOLPROP_NO_EXAMPLES
 #        "On Darwin systems, compile and link with -std=libc++ instead of the default -std=libstdc++"
 #        ON)
 
+# Force C++11 since lambdas are used in CPStrings.h
+set(CMAKE_CXX_STANDARD 11)
+
```

Comment:
I had to force this for mac

Reply:
I think that is fine.  I'm looking forward to C++11.